### ISSUE TITLE
Unable to connect

### ISSUE THREAD

**Issue by ghost** [thingsSDK/thingssdk-cli#47]
Hello, I'm unable to connect

```
Treeshaking code...
Sending code to device - /dev/tty.wchusbserial1410 @ 115200 baud...
espruino:  Espruino Command-line Tool 0.1.6
-----------------------------------

espruino:  Connecting to '/dev/tty.wchusbserial1410'
espruino:  Port "/dev/tty.wchusbserial1410" not found
espruino:  Unable to connect!
Exited with status 0
```

Node v9.5.0 on Mojave (Mac), Espruino Web IDE works fine with that port

**Comment by chalkers** [comment]
Are you still connected in the web ide at the time? Also is the port in the web ide exactly the same as the one in the CLI?

Run `thingssdk devices` to double check. Each board may have a slightly different port name, the port is cached in project to project.

**Comment by ghost** [comment]
> Are you still connected in the web ide at the time? Also is the port in the web ide exactly the same as the one in the CLI?

Nope the web ide is not running , and yes the port is actually the good one. I've run the configurator once again, this is the result 

```
{
  "devices": {
    "/dev/tty.wchusbserial1410": {
      "baud_rate": 115200,
      "runtime": "espruino"
    }
  }
}
```

Just for fun I've updated the Espruino CLI to 0.1.21 and now it connects well, however it gets stuck into node environment

![2018-12-08 22 52 24](https://user-images.githubusercontent.com/1085151/49691132-26101d80-fb3c-11e8-9d5a-278dfdc9815e.gif)
 

However I can write any file with `espruino -p /dev/tty.wchusbserial1410 test.js -b 115200` and it works fine.

**Comment by chalkers** [comment]
`dev` deploys the code without saving, and then runs the repl for you to debug or do other inline commands.

For example you can type `1+1`, look at global variables or whatever.

If you run `npm run deploy` it uploads the code and saves it.

**Comment by ghost** [comment]
Yeah, but the problem is that neither `dev` or `deploy` work. Or am I missing something about your comment ?
